packages/apps/human-app/frontend/src/pages/worker/jobs/components/available-jobs/desktop/available-jobs-table.tsx

```diff
@@ -36,9 +36,12 @@ export type AvailableJobsTableData = AvailableJob & {
   };
 };
 
-const getColumns = (callbacks: {
-  assignJob: (data: AssignJobBody) => undefined;
-}): MRT_ColumnDef<AvailableJob>[] => {
+const getColumns = (
+  callbacks: {
+    assignJob: (data: AssignJobBody) => undefined;
+  },
+  disableButtons: boolean
```

Comment:
It would be cleaner if we pass just object to `getColumns`, something like
```
{
  handleAssignJobClick: (data: AssignJobBody) => undefined,
  disableButtons: boolean
}
```


This way in place where you call `getColumns` it's clear what to expect and you don't have to check the signature of that function